css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  justify-content: center;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  width: 60%;
+}
+
+.summary-dk {
```

Comment:
we don't want to use auto width anywhere is this exercise, instead, make sure to set the appropriate widths as needed to the column classes.